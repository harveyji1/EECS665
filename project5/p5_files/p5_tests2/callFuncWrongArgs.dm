fn : (a : int) int {
    return a + 1;
}

main : () void {
    fn(1, 2);
}
