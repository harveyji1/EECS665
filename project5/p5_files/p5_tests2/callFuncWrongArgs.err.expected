FATAL [6,5]-[6,13]: Function call with wrong number of args
Type Analysis Failed
